README.md

```diff
@@ -100,7 +106,8 @@ Select 1:
 What is the difference between a fork and a clone?
 
 ```text
-Your answer here.
+A fork creates a remote copy of a repo on your github profile.
+A clone creates a local copy of a remote repo.
```

Comment:
:+1:  A clone is a version of someone else's repository downloaded to your computer. A fork is a copy of someone else's repository, not necessarily downloaded to your computer. You can clone a fork; you can't fork a clone.